gensim/models/word2vec.py

```diff
@@ -100,15 +101,16 @@
 from types import GeneratorType
 
 logger = logging.getLogger(__name__)
-
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s')
 
 try:
     from gensim.models.word2vec_inner import train_batch_sg, train_batch_cbow
     from gensim.models.word2vec_inner import score_sentence_sg, score_sentence_cbow
     from gensim.models.word2vec_inner import FAST_VERSION, MAX_WORDS_IN_BATCH
+    logger.debug('Fast version of {0} is being used'.format(__name__))
 except ImportError:
     # failed... fall back to plain numpy (20-80x slower training than the above)
-    # Can't log here because logger is usually not configured at import time
+    logger.warning('Slow version of {0} is being used'.format(__name__))
```

Comment:
Does this work fine? (the removed comment seems to indicate otherwise, just making sure)